models/topics.js

```diff
@@ -54,4 +55,43 @@ exports.selectCommentsByArticleId = (article_id) => {
     });
 };
 
+exports.addComment = (article_id, username, body) => {
+    if (!username || !body) {
+        return Promise.reject({ status: 400, msg: 'missing object properties' })
+    } else {
+        return db
+        .query(`INSERT INTO users (username, name) VALUES ($1, $2) RETURNING*;`, [username, 'name'])
```

Comment:
I think this is how you are managing to pass the test! This endpoint should certainly NOT be adding in any users. But it now makes sense - this MUST be removed 👍